config/routes.rb

```diff
@@ -259,12 +260,15 @@
 
   # path for orphaned sites
   get 'sites/orphans' => 'sites#orphans'
-  post 'sites/orphans/filter' => 'sites#orphans'
+  match 'sites/orphans/filter' => 'sites#orphans', via: [:get, :post], defaults: { format: 'json' }
 
   # shallow path to sites
 
   resources :sites, except: [:edit], defaults: { format: 'json' }, as: 'shallow_site'
 
+  match 'regions/filter' => 'regions#filter', via: [:get, :post], defaults: { format: 'json' }
+  resources :regions, except: [:edit], defaults: { format: 'json' }, as: 'shallow_region'
```

Comment:
`:edit` is the before step for an update, like how `:new` is the step before `:create`. They are part of the standard rails actions for a controller. Edit is the view shown that allows a user to change details of an object. Unlike `:new:` however, `:edit` does not have a meaningful API version for that action. For the API we just `:show` then `:update`. We could include `:edit` as a synonym for `:show`, since in the API context there is no semantic difference, but that is unnecessary functionality which would only be added so that we had the _standard rails actions_ set up...